### PR TITLE
Add page not found banner

### PR DESCRIPTION
**Pull request by Tguntenaar**
_Resolves [#970](https://github.com/NEAR-DevHub/neardevhub-bos/issues/970)_

### CODE REVIEW

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM

**Comment by Megha-Dev-19** [comment]
@race-of-sloths include

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 3

File: instances/infrastructure-committee.near/widget/components/proposals/Proposal.jsx [hunk 1]
```diff
@@ -259,6 +259,16 @@ const LinkProfile = ({ account, children }) => {
 const stepsArray = [1, 2, 3, 4, 5];
 
 const { id, timestamp } = props;
+
+if (id === undefined) {
+  return (
```
[the remark this answers is not in the document]
Comment (Tguntenaar) [reply instances/infrastructure-committee.near/widget/components/proposals/Proposal.jsx:264]: Yes done

**Pull request merged**